tests/acceptance/features/bootstrap/CalDavContext.php

```diff
@@ -129,6 +129,38 @@ public function theCalDavHttpStatusCodeShouldBe($expectedStatus) {
 		);
 	}
 
+	/**
+	 * @Then user :user should get response for calendar :calendar using WebDAV API
```

Comment:
```suggestion
	 * @Then user :user should have received a response for calendar :calendar using WebDAV API
```
try to be in past perfect for `then` steps